CodeEdit/Breadcrumbs/BreadcrumbsView.swift

```diff
@@ -65,11 +65,14 @@ struct BreadcrumbsView: View {
 	}
 
     private func fileInfo(_ file: WorkspaceClient.FileItem) {
-		guard let projName = workspace.fileURL?.lastPathComponent,
-			  var components = file.url.pathComponents.split(separator: projName).last else { return }
-		components.removeLast()
+		guard let projURL = workspace.fileURL else { return }
+		let components = file.url.path
+			.replacingOccurrences(of: projURL.path, with: "")
```

Comment:
Let's start adding some comments here that explains why we are splitting, mapping and dropping. Something short that will help future us understanding the decision! Great Work!

Reply:
😀 I'll do that later.